stairway/src/test/java/bio/terra/stairway/FlightStateClassTest.java

```diff
@@ -38,6 +39,8 @@ public void setup() {
 
     timestamp = Instant.now();
 
+    className = "bio.terra.stairway.FooFlight";
```

Comment:
This could be `private static final`

Reply:
ah good call.  Update that and the `bad` var to be static final